src/backend/opencl/CMakeLists.txt

```diff
@@ -12,46 +12,34 @@ ENDIF()
 ADD_DEFINITIONS(-DAF_OPENCL
                 -D__CL_ENABLE_EXCEPTIONS)
 
-# clBLAS and clFFT create packages when their `make install` step is executed,
-# rather than installing to system directories. Thus we have special find
-# and install methods for these libraries.
-FIND_PACKAGE(CLBLAS REQUIRED)
+OPTION(USE_SYSTEM_CLBLAS "Use system clBLAS" OFF)
+IF(USE_SYSTEM_CLBLAS)
+  FIND_PACKAGE(clBLAS REQUIRED)
+ELSE()
+  INCLUDE("${CMAKE_MODULE_PATH}/build_clBLAS.cmake")
+ENDIF()
 INCLUDE_DIRECTORIES(${CLBLAS_INCLUDE_DIRS})
 LINK_DIRECTORIES(${CLBLAS_LIBRARY_DIR})
-# Custom install command
-INSTALL(DIRECTORY "${CLBLAS_PACKAGE_DIR}/include/" DESTINATION include
-    COMPONENT headers)
-IF(UNIX)
-    FILE(GLOB CLBLAS_LIBS "${CLBLAS_PACKAGE_DIR}/lib64/*${CMAKE_SHARED_LIBRARY_SUFFIX}*")
-ELSEIF(WIN32)
-    FILE(GLOB CLBLAS_LIB "${CLBLAS_PACKAGE_DIR}/lib64/import/*${CMAKE_STATIC_LIBRARY_SUFFIX}")
-    FILE(GLOB CLBLAS_DLL "${CLBLAS_PACKAGE_DIR}/bin/*${CMAKE_SHARED_LIBRARY_SUFFIX}")
-  LIST(APPEND CLBLAS_LIBS ${CLBLAS_LIB} ${CLBLAS_DLL})
-ENDIF(UNIX)
-INSTALL(FILES ${CLBLAS_LIBS} DESTINATION lib COMPONENT libraries)
-
-FIND_PACKAGE(clFFT REQUIRED)
+
+OPTION(USE_SYSTEM_CLFFT "Use system clFFT" OFF)
+IF(USE_SYSTEM_CLFFT)
+  FIND_PACKAGE(clFFT REQUIRED)
+ELSE()
+  INCLUDE("${CMAKE_MODULE_PATH}/build_clFFT.cmake")
```

Comment:
:-(

We really shouldn't have to do this - I'll think a bit more about https://github.com/clMathLibraries/clFFT/pull/75 to see if it can be done directly there.